src/components/about/partners/partners-data.js

```diff
@@ -22,6 +22,29 @@ import fieldMuseumLogo from 'logos/field-museum-logo_2018.png';
 import gorongosaLogo from 'logos/gorongosa_np_logo.png';
 import inaturalistLogo from 'logos/inaturalist.png';
 
+export const texts = {
+  partners: {
+    title: 'Half-Earth Project Map - Core Team',
+    content: [
+      'The Half-Earth Project is an initiative of the E.O. Wilson Biodiversity Foundation. Map of Life utilizes geospatial species distribution data and analytics to guide where we have the best opportunity to conserve the most species. Vizzuality brings this information to life.',
+      `Click here to know more about the core team of Half-Earth:
+      <a target="_blank" href="https://www.half-earthproject.org/half-earth-project-team-members/">www.half-earthproject.org/half-earth-project-team-members</a>`
```

Comment:
What do you think about displaying the link in a different line always as in the design? Maybe just wrapping it on a div or using a class. This is how it shows on my screen:

![image](https://user-images.githubusercontent.com/9701591/95448139-21e4ff00-0963-11eb-92d3-71b39c500f02.png)

Reply:
done :+1: 
Thanks!